### PR TITLE
docsp-28026 - 1.11.2 patch updates

### PR DESCRIPTION
**Pull request by mongoKart**
[Release Notes](https://github.com/mongodb/mongo-go-driver/releases/tag/v1.11.2)

# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-28026
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/docsp-28026-go-updates/whats-new/

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -33,6 +33,15 @@ Learn what's new in:
 What's New in 1.11
 --------------------
 
+.. important:: Upgrade to Version 1.11.2 or Higher 
+
+  The 1.11.2 Go driver contains the following bug fixes:
+  
+  - ``bson.Raw.String`` outputs the correct Extended JSON type for timestamps when
```
Comment (rustagir) [inline source/whats-new.txt:40]: Suggestion: from what I found in the API docs, this is referring to a method. I suggest including the parentheses to make this clearer, or going even further and describing it as a method

```suggestion
  - ``bson.Raw.String()`` outputs the correct Extended JSON type for timestamps when 
```
or
```suggestion
  - The ``bson.Raw.String()`` method outputs the correct Extended JSON type for timestamps when 
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -33,6 +33,15 @@ Learn what's new in:
 What's New in 1.11
 --------------------
 
+.. important:: Upgrade to Version 1.11.2 or Higher 
+
+  The 1.11.2 Go driver contains the following bug fixes:
+  
+  - ``bson.Raw.String`` outputs the correct Extended JSON type for timestamps when 
+    the BSON document contains a ``UTC Timestamp`` field.
```
Comment (rustagir) [inline source/whats-new.txt:41]: Issue: I'm not sure that UTC Timestamp is the actual type name, could it be just [date](https://www.mongodb.com/docs/manual/reference/bson-types/#date)?

Suggestion: UTC Timestamp not monospaced

**Review by rustagir — APPROVED** [review]
LGTM with suggestions, thought they were small enough that I didn't want to block

File: source/whats-new.txt (hunk 2 above)
Comment (mongoKart) [reply source/whats-new.txt:41]: Makes sense. Looks like [the BSON type is "Timestamp"](https://github.com/mongodb/specifications/blob/master/source/extended-json.rst#conversion-table) but removing monospace seems like the safest option.

**Pull request merged**